docs/release-info/release-notes.md

```diff
@@ -28,7 +28,10 @@ Apache Druid 28.0.1 is a patch release that fixes some issues in the Druid 28.0.
 
 ### Notable improvements
```

Comment:
These should be bug fixes IMO. Many of these were regressions introduced in this release or something that would have likely become more common due to the changes in this release.